test/unittests/commands/validate_data/sparql/test_cardinality_shacl.py

```diff
@@ -19,7 +20,7 @@
 class TestCheckTripleNumbersOnto:
     def test_nodeshape(self, onto_graph: Graph) -> None:
         result = _construct_resource_nodeshape(onto_graph)
-        number_of_resource_classes = 6
+        number_of_resource_classes = 12
```

Comment:
I updated the `onto.ttl` file, which is the cause of this.
I haven't done this after including a second onto. Which is why that is also added now.